source/guides/administrator.rst

```diff
@@ -13,6 +13,7 @@ Use the following materials as a template to help you plan and document the impl
    :glob:
 
    /getting-started/enterprise-roll-out-checklist.rst
+   /deployment/enterprise-deployment-guide.rst
```

Comment:
Let's add it to the new "deployment guides" section in "training and education"  https://docs.mattermost.com/guides/education.html#deployment-guides